### PR TITLE
Add error handling

### PR DESCRIPTION
**Pull request by tommi-t-nguyen**
What (if any) features are you implementing?
-error handling
What (if anything) did you refactor?
-n/a
Were there any issues that arose?
-n/a
Is there anything that you need from your teammate?
-let get this css going
Any other comments, questions, or concerns?
-n/a

### CODE REVIEW

**Comment by susannaopal** [comment]
Now that we have error handlings et up -- ready to keep moving, thanks for your review, Jani!

File: src/App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ class App extends Component {
   componentDidMount(){
     fetchAllMovies()
     .then(data => this.setState({movies: data.movies}))
+    .catch((error) => this.setState({view: 'error'}))
   }
 
   switchView = (id, views) => {
```
Comment (hannahhch) [inline src/App.js:23]: Oh, I love the naming for this function! Make sure the code inside only has to do with switching views. Have your App component return the main, and h1, with your switchViews called inside.

File: src/Components/MovieDetail/MovieDetail.js [hunk 2]
```diff
@@ -7,22 +7,25 @@ class MovieDetail extends Component {
...
+
+
   render() {
     return (
-    <>
-      <img src={this.state.detail.backdrop_path} alt='poster'width="90%"/>
-      <p>{this.state.detail.title}</p>
-    </>
+<>
+    {this.state.error && <h1>Sorry we're having techincal difficulty. Please try again later.</h1>}
```
Comment (hannahhch) [inline src/Components/MovieDetail/MovieDetail.js:25]: Might be worth making a reusable Error component!

**Review by hannahhch — COMMENTED** [review]
Great job so far!

**Pull request merged**